src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -0,0 +1,130 @@
+# NTFS to Netex France conversion
+
+Version of the implementation: `1.0`
+
+## Introduction
+This document describes how a [NTFS feed](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md) is transformed into a Netex profil France feed in Navitia Transit Model.
+
+The resulting ZIP archive is composed of:
+- a `arrets.xml` file containing the description of all stops (Quays and StopPlaces)
+- a `correspondances.xml` file containing all transfers between stops
+- a folder for each network containing
+  + 
+
+## File headers
+
+Each Netex file starts with a header containing some informations generated by the programm: 
+- `/PublicationDelivery/@version`: **x.y:FR-NETEX_nnnn-a.b-c** with
+  + `x.y`:  the version of Netex XSD of Netex (actually 1.09)
+  + `nnnn`:identification of the profile ("FRANCE", "COMMUN", "ARRET",
+"LIGNE", "RESEAU", "HORAIRE", "CALENDRIER" or "TARIF") 
+  + `a.b` is the profile version (2.1)
+  + `c` is the number of the local implementation (containing only integers and `.`). See at the top of the document.
+- `/PublicationDelivery/PublicationTimestamp`: generation date of the Netex Feed using ISO8601,
+- `/PublicationDelivery/ParticipantRef`: The value of the corresponding parameter to the conversion action.
```

Comment:
Maybe a section at the beginning of the document concerning the input parameters would be useful so you can refer to them (because I feel there is going to be more than one parameter to this connector).

```md
## Input parameters

- `participant`: blahblahblah
```

Reply:
Nice idea :+1:  I've added one section at the top of the document.